Sources/CartonKit/Server/Server.swift

```diff
@@ -178,7 +178,7 @@ public actor Server {
         Task { try await self.onChange(changes, configuration) }
       }
       try watcher?.start()
-	}
+  }
```

Comment:
sorry, I misaligned the previous suggestion 🤦‍♂️
```suggestion
    }
```